Sofa/framework/Config/CMakeLists.txt

```diff
@@ -45,13 +46,11 @@ endif()
 # nedd to generate the double related code.
 if(${SOFA_FLOATING_POINT_TYPE} STREQUAL float)
     set(SOFA_WITH_FLOAT 1)
+    set(SOFA_WITH_DOUBLE 0)
 endif()
 if(${SOFA_FLOATING_POINT_TYPE} STREQUAL double)
     set(SOFA_WITH_DOUBLE 1)
-endif()
-if(${SOFA_FLOATING_POINT_TYPE} STREQUAL both)
-    set(SOFA_WITH_FLOAT 1)
-    set(SOFA_WITH_DOUBLE 1)
+    set(SOFA_WITH_FLOAT 0)
```

Comment:
Are `SOFA_WITH_FLOAT` and `SOFA_WITH_DOUBLE` still used?

Reply:
No idea, seems not